README.md

```diff
@@ -74,105 +83,115 @@ type SimpleOrder record {|
 
 ### Reading EDI files
 
-Below code reads the edi-sample1.edi into a json variable named "orderData" and then convert the orderData json to the generated record "SimpleOrder". Once EDI documents are mapped to the SimpleOrder record, any attribute in the EDI can be accessed using record's fields as shown in the example code below.
+Generated ```fromEdiString``` function can be used to read EDI text files into the generated Ballerina record as shown below. Note that any data item in the EDI can be accessed using record's fields as shown in the example code.
 
 ````ballerina
 import ballerina/io;
-import balarina/edi;
 
 public function main() returns error? {
-	edi:EDISchema schema = check edi:getSchema(check io:fileReadJson("resources/edi-schema1.json"));
     string ediText = check io:fileReadString("resources/edi-sample1.edi");
-    json orderData = check edi:read(ediText, schema);
-    io:println(orderData.toJsonString());
-
-    SimpleOrder order1 = check orderData.cloneWithType(SimpleOrder);
+    SimpleOrder order1 = check hmartOrder:fromEdiString(ediText);
```

Comment:
```suggestion
    SimpleOrder order = check hmartOrder:fromEdiString(ediText);
```

Reply:
I guess having order1 (for example 1) and order2 (for exampe 2) is better than having order and order1. Also, "order" is a keyword.